pub/index.html

```diff
@@ -14,11 +14,99 @@
 
         <link rel="stylesheet" href="css/normalize.css">
         <link rel="stylesheet" href="css/main.css">
+        <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
+        <link href='http://fonts.googleapis.com/css?family=Oswald|Open+Sans' rel='stylesheet' type='text/css'>
         <script src="js/vendor/modernizr-2.6.2.min.js"></script>
     </head>
     <body>
-
-
+        <div class="page">
+            <header>
+                <div class="top-container">
+                    <div class="search">
+                        <a href="">About</a>
+                        <a href="">Search</a>
+                        <a class="fa fa-search"></a>
+                    </div>
+                        <div class="logo">
+                            <img src="../pub/wave.png">
+                            <span class="hidden">SURF & PADDLE CO. BLOG</span>
+                        </div>
+                </div>    
+                <h1>SURF & PADDLE CO.</h1>
```

Comment:
Github (and `git`) highlights this red to let you know it's a no-no. Use `&amp;` instead of a literal `&` inside of HTML, just like you should use `&gt;` instead of `<`... See [MDN](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Introduction#Named_character_references)